transactor/src/main/scala/io/mediachain/transactor/JsonSerialization.scala

```diff
@@ -0,0 +1,64 @@
+package io.mediachain.transactor
+
+import io.mediachain.transactor.Types._
+import org.json4s._
+
+case class IPLDReference(link: String) extends Reference
```

Comment:
yeah, I think I'll remove this type from this branch entirely.  I was using it before you merged in the DummyReference, which will do for now.  We can more fully work out the IPFS references when we tackle the IPFS datastore